### ISSUE TITLE
[Feature Request] Force Email Verify

### ISSUE THREAD

**Issue by inegan** [Katamaze/WHMCS-Action-Hook-Factory#29]
it will be  cool if you can make Force Email Verify

when client sign up even i enabled " Send email confirm " at my whmcs Settings

Client still can place order and get The Product With fake email

even worse there clients use a fake email on my site and when pay with paypal use another one . and i will not able to give paypal a proof that was same pesron

it will be nice if you can make Hook that not let user Place orders only When He Verify His Account First!

Thanks For all this great work!

**Comment by whattheserver** [comment]
Yeah this would be an amazing hook. It is weird whmcs doesn't already do this and this would save alot of those hassles.
